style.css

```diff
@@ -88,6 +90,219 @@ a {
   padding: 5px;
 }
 
+.categories-list li {
+  font-weight: 400;
+  font-size: 1.2rem;
+  line-height: 3rem;
+  color: #fff;
+}
+
 .social-media ul li {
   padding: 5px;
 }
+
+#recentWorks {
+  display: grid;
+  height: auto;
+  grid-template-columns: 100%;
+}
+
+#recentWorks h2 {
+  color: #1c1a19;
+  width: 300px;
+  font-weight: 400;
+  font-size: 36px;
+  border-bottom: 1px solid #212121;
+  align-items: center;
+  padding: 12px;
+}
+
+.imgHeader {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 2vh;
+}
+
+.title {
+  color: #091e42;
+  font-size: 29px;
+}
+
+.introInfo {
+  color: #091e42;
+  font-size: 15px;
+}
+
+#works {
+  margin: 3.5vh 11vw;
+}
+
+.buttonGroup {
+  display: flex;
+}
+
+.bigButton {
+  display: flex;
+  background: #ff6b00;
+  color: #fff;
+  align-items: center;
+  padding: 10px;
+  justify-content: center;
+  width: 110px;
+  margin-top: 3px;
+}
+
+.button {
+  border: 1px solid #8993a4;
+  background: #fff;
+  padding: 10px 12px;
+  color: #091e42;
+  margin: 3px;
+}
```

Comment:
- [ ] As stated in the project requirement you are required to Implement the button interactions (enable, hover, etc.). So Kindly work on it.

![Capture](https://user-images.githubusercontent.com/93598189/205006918-3fb2cf00-48a4-43a5-bbe2-8ae4034657c2.PNG)

Reply:
This is mobile version not desktop. Why are you giving me comments on desktop view when we are working on mobile version. please check the template very well.

Reply:
Hello @til2to in the project requirement you required to implement  button interactions (enable, hover, etc.) , Please check the project requirement.